Makie/src/makielayout/types.jl

```diff
@@ -297,6 +297,33 @@ Axis(fig_or_scene; palette = nothing, kwargs...)
         """
         dim2_conversion = nothing
 
+        "Controls whether the x dim_convert is shown in ticklabels."
+        x_unit_in_ticklabel::Union{Bool, Automatic} = automatic
+        "Controls whether the y dim_convert is shown in ticklabels."
+        y_unit_in_ticklabel::Union{Bool, Automatic} = automatic
+        "Controls whether the x dim_convert is shown in the xlabel."
+        x_unit_in_label::Union{Bool, Automatic} = automatic
+        "Controls whether the y dim_convert is shown in the ylabel."
+        y_unit_in_label::Union{Bool, Automatic} = automatic
+        """
+        Formatter for the xlabel suffix generated from dim_converts. Can be a
+        Format.jl format string or a callback function acting acting on the
+        string or rich text generated from the dim convert.
+        Can also be a plain String replacing an active dim_convert label.
+        """
+        xlabel_suffix = "[{}]"
+        """
+        Formatter for the ylabel suffix generated from dim_converts. Can be a
+        Format.jl format string or a callback function acting acting on the
+        string or rich text generated from the dim convert.
+        Can also be a plain String replacing an active dim_convert label.
+        """
+        ylabel_suffix = "[{}]"
+        "Switches between short and long x units, e.g. \"s\" vs \"Second\""
+        use_short_x_units::Bool = true
+        "Switches between short and long y units, e.g. \"s\" vs \"Second\""
+        use_short_y_units::Bool = true
```

Comment:
These probably need some renaming to fix with Axis attributes better? I guess just remove all the `_`?